services/src/main/java/org/exoplatform/webconferencing/notification/plugin/CallRecordingPlugin.java

```diff
@@ -0,0 +1,60 @@
+package org.exoplatform.webconferencing.notification.plugin;
+
+import org.exoplatform.commons.api.notification.NotificationContext;
+import org.exoplatform.commons.api.notification.model.NotificationInfo;
+import org.exoplatform.commons.api.notification.plugin.BaseNotificationPlugin;
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.webconferencing.WebConferencingService;
+import org.exoplatform.webconferencing.notification.utils.NotificationArguments;
+
+import java.util.List;
+
+public class CallRecordingPlugin extends BaseNotificationPlugin {
+
+    public static final  String ID  = "CallRecordingPlugin";
+
+    public static final String RECEIVER = null;
+
+    public CallRecordingPlugin(InitParams initParams) {
+        super(initParams);
+    }
+
+    @Override
+    public String getId() {
+        return ID;
+    }
+
+    @Override
+    public boolean isValid(NotificationContext notificationContext) {
+        return true;
+    }
+
+    @Override
+    protected NotificationInfo makeNotification(NotificationContext notificationContext) {
+        String recordingStatus= notificationContext.value(NotificationArguments.RECORDING_STATUS);
+        String fileUrl = notificationContext.value(NotificationArguments.RECORDED_FILE_URL);
+        String fileName = notificationContext.value((NotificationArguments.FILE_NAME));
+        String callType = notificationContext.value((NotificationArguments.CALL_TYPE));
+
+        if(callType.equals(WebConferencingService.OWNER_TYPE_SPACE) || callType.equals(WebConferencingService.OWNER_TYPE_SPACEEVENT)) {
+            List<String> participants = notificationContext.value(NotificationArguments.CALL_PARTICIPANTS);
+            return NotificationInfo.instance()
+                    .to(participants)
+                    .with(NotificationArguments.RECORDING_STATUS.getKey(), recordingStatus)
+                    .with(NotificationArguments.FILE_NAME.getKey(), fileName)
+                    .with(NotificationArguments.RECORDED_FILE_URL.getKey(), fileUrl)
+                    .key(getKey())
+                    .end();
+        } else {
+            String receiver = notificationContext.value(NotificationArguments.RECEIVER);
+            return NotificationInfo.instance()
+                    .to(receiver)
+                    .with(NotificationArguments.RECORDING_STATUS.getKey(), recordingStatus)
+                    .with(NotificationArguments.FILE_NAME.getKey(), fileName)
+                    .with(NotificationArguments.RECORDED_FILE_URL.getKey(), fileUrl)
+                    .key(getId())
+                    .end();
+        }
+
```

Comment:
Can we imaging remove the if here ? 
In case of type = space of spaceevent, participants is a list of user, and in case of room call or 1v1 call, can we have a list of one participant ?
It could make the code cleaner here.

WDYT ?